### ISSUE TITLE
False positive for echo ( $var

### ISSUE THREAD

**Issue by drinkmaker** [envato/envato-theme-check#32]
Hello,

Here is warning:
`WARNING: Found echo ( $ in the file...Possible data validation issues found. All dynamic data must be correctly escaped for the context where it is rendered.`

Here is code:
```php
<article id="comment-<?php comment_ID(); ?>" class="uk-comment <?php echo ( $comment->user_id > 0 ) ? 'uk-comment-primary' : ''; ?>">
```

As you can see, here we have false positive because of short syntax "if".

**Comment by scottparry** [comment]
Yes, that won't fail when reviewers check. Anything that is a WARNING is for reviewers to check due to possible false positives.
